summertime/model/single_doc/bart_model.py

```diff
@@ -22,6 +22,8 @@ def __init__(self, device="cpu"):
     def summarize(self, corpus, queries=None):
         self.assert_summ_input_type(corpus, queries)
 
+        self.assert_summ_input_language(corpus, queries)
```

Comment:
I have made a comment about this in #96, after the refactoring on that branch, let's merge it to main and pull main for this branch so it's fixed here automatically as well.